test/data/ch4_ion.yaml

```diff
@@ -0,0 +1,142 @@
+units: {length: cm, quantity: mol, activation-energy: cal/mol}
+
+phases:
+- name: gas
+  thermo: ideal-gas
+  transport: ionized-gas
+  species: [O2, H2O, CO2, N2, H3O+, E, O2-]
+  state: {T: 300, P: 1 atm}
+
+species:
+- name: O2
+  composition: {O: 2}
+  thermo:
+    model: NASA7
+    temperature-ranges: [200, 1000, 3500]
+    data:
+    - [3.78245636E+00, -2.99673416E-03, 9.84730201E-06, -9.68129509E-09,
```

Comment:
I presume these go high temperature then low temperature, to match the original fixed format spec. However, can we be explicit about which is which? We might as well take advantage of the format to be a little more clear.

```yaml
data:
  high_T: [3.78...]
  low_T: [3.28...]
```

Reply:
Actually, they're low then high, to match the temperature ranges - I think the original format is bonkers. Using `high-T` and `low-T` labels doesn't generalize very well to the case where only one polynomial is provided, or to the NASA9 format, where we support an arbitrary number of temperature regions.

Reply:
The original format may be bonkers, but if we're changing something then we need to document it (I haven't looked to see if that has changed).

Reply:
It is described in the API docs. See [here](http://gir.mit.edu/~speth/cantera-yaml-api/yaml/species.html#sec-yaml-nasa7).

Reply:
Also, while the original fixed-width NASA format has the high-temperature polynomial first, files converted to CTI or Cantera XML have always had the low temperature polynomial first.